addons/l10n_sa_edi/tests/common.py

```diff
@@ -214,7 +220,7 @@ def _create_invoice(self, **kwargs):
             ],
         }
         move = self.env['account.move'].create(vals)
-        move.state = 'posted'
+        move.state = kwargs.get('state') or 'posted'
```

Comment:
oO 
What is that? What kind of test would define the state of the move to posted without actually posting it?
That's horrifying